### ISSUE TITLE
Update README with database setup instructions for sessions

### ISSUE THREAD

**Issue by isaacdontjelindell** [LutherSrProject/TapVote#199]
I believe all that really needs to be done is install pgtap? Our migrate scripts should do the rest...

**Comment by etingel** [comment]
Yes, exactly. There are optional environment variables for a session user/db if someone wants to lock things down.
